aperturedb/ParallelQuery.py

```diff
@@ -9,22 +9,22 @@
 
 from aperturedb.DaskManager import DaskManager
 from aperturedb.Connector import Connector
-from aperturedb.types import Query, Blobs, CommandResponses
+from aperturedb.types import Commands, Blobs, CommandResponses
```

Comment:
Nice, how did this not manifest while running locally.

Reply:
Unclear.  Let's talk about local Python testing when we get a chance.